android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -372,39 +365,57 @@ public View getFormDataView(String address) {
     @Override
     public JSONObject getObjectUsingAddress(String[] address, boolean popup) throws JSONException {
         if (address != null && address.length > 1) {
-            if (RuleConstant.RULES_ENGINE.equals(address[0])) {
+            if (RuleConstant.RULES_ENGINE.equals(address[0]) && !address[1].endsWith(".yml")) {
```

Comment:
Is there a more unique check to verify that this uses dynamic relevance other than checking that this does not contain a `.yml` reference?

Checking for `.yml` references may cause unexpected behavior if another implementation is needed (or maybe already exists) which does not also use `.yml`.

Reply:
Got any suggestions